Automattic/vip-go-mu-plugins
# Add wp vip cache purge-url subcommand

## Description

Currently, we have a rather nuclear option to purge the whole site cache from Varnish. It is needed sometimes, but more often than not all is really needed is to clear just one or two URLs.

This PR addresses that by adding `wp vip cache purge-url` command that does exactly that.

Additionally, this PR introduces a confirmation prompt when purging the cache for the whole site.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Checkout the PR
2. Sandbox a site
3. Hit a URL on sandbox, make sure it got cached.
4. Run `wp vip cache purge-url <URL>`
5.  Hit the URL again, make sure `x-cache: miss` header is present.
